tests/lambda_endpoints/test_batch_starter.py

```diff
@@ -1,293 +1,231 @@
+        instrument="hit",
+        data_level="l0",
+        descriptor="science",
+        start_date=datetime(2024, 1, 1),
+        end_date=datetime(2024, 1, 2),
+        version="v00-01",
+        extension=".cdf",
+        status_tracking_id=1,  # Assuming a valid ID from 'status_tracking' table
+    )
+    with Session(db.get_engine()) as session:
+        session.add(test_record_1)
+        session.add(test_record_2)
+        session.commit()
+
+    return session
 
 
 @pytest.fixture()
-def database(postgresql):
-    """Populate test database."""
-
-    cursor = postgresql.cursor()
-    cursor.execute("CREATE SCHEMA IF NOT EXISTS sdc;")
-
-    # Drop the table if it exists, to start with a fresh table
-    cursor.execute("DROP TABLE IF EXISTS sdc.codicehi;")
-    # TODO: sync with actual database schema once it is created
-    sql_command = """
-    CREATE TABLE sdc.codicehi (
-        -- Primary key
-        id SERIAL PRIMARY KEY,
-
-        -- Basic columns
-        filename TEXT UNIQUE NOT NULL,
-        instrument TEXT NOT NULL,
-        version INTEGER NOT NULL,
-        level TEXT NOT NULL,
-        mode TEXT,
-        date TIMESTAMP WITH TIME ZONE DEFAULT NULL,
-        ingested TIMESTAMP WITH TIME ZONE DEFAULT (now() AT TIME ZONE 'UTC'),
-        mag_id INTEGER,
-        spice_id INTEGER,
-        parent_codicehi_id INTEGER,
-        pointing_id INTEGER
-    );
-    """
-
-    cursor.execute(sql_command)
-
-    hardcoded_data = [
-        {
-            "id": 2,
-            "filename": "imap_codicehi_l1b_20230531_v01.cdf",
-            "instrument": "codicehi",
-            "version": 1,
-            "level": "l1b",
-            "mode": None,
-            "date": "2023-05-31 14:45:00+03",
-            "ingested": "2023-06-02 14:45:00+06",
-            "mag_id": None,
-            "codicelo_id": None,
-            "spice_id": None,
-            "parent_codicehi_id": 1,
-            "pointing_id": 1,
-        },
-        {
-            "id": 4,
-            "filename": "imap_codicehi_l3a_20230602_v01.cdf",
-            "instrument": "codicehi",
-            "version": 1,
-            "level": "l3a",
-            "mode": None,
-            "date": "2023-06-02 14:45:00+03",
-            "ingested": "2023-06-02 14:45:00+10",
-            "mag_id": None,
-            "codicelo_id": None,
-            "spice_id": None,
-            "parent_codicehi_id": 3,
-            "pointing_id": 1,
-        },
-        {
-            "id": 5,
-            "filename": "imap_codicehi_l3b_20230531_v01.cdf",
-            "instrument": "codicehi",
-            "version": 1,
-            "level": "l3b",
-            "mode": None,
-            "date": "2023-05-31 14:45:00+03",
-            "ingested": "2023-06-02 14:45:00+11",
-            "mag_id": 4,
-            "codicelo_id": None,
-            "spice_id": None,
-            "parent_codicehi_id": 4,
-            "pointing_id": 1,
-        },
-    ]
+def _aws_credentials():
```

Comment:
`conftest.py` in this directory already takes care of this. You can remove this and call it in functions that needs it.